best-first-cv/script.whizzml

```diff
@@ -528,15 +675,36 @@
                                                      {}
                                                      true)) all-reqs)
         ;;vs (map (lambda (ev) (ev ["result" "model" "average_phi"] 0)) cvs)
-        vs cvs
+        ;; Obtains phi-stdev from each cross validation
+        vs (map (lambda (ev) (ev ["phi-stdev"])) cvs)
         value-map (make-map potentials vs)
         max-val (get-max vs)
-        choose-best (lambda (id) (and (= max-val (value-map id)) id)))
+        choose-best (lambda (id) (and (= max-val (value-map id)) id))
+        phi (some (lambda (n) (if (= (n ["phi-stdev"]) max-val)
+                                  (n ["phi"]) 
+                                  false)) 
```

Comment:
`if`s returning booleans are almost always clunky.  better: `(and (= (max-val (n "phi-stdev")) (n "phi"))` or `(when (= max-val (n "phi-stdev")) (n "phi"))`